### PR TITLE
Add postgresql migration

### PR DESCRIPTION
**Pull request by Josipmrden**
### Description

Postgresql migration module has been added.

### Pull request type

- [ ] Bugfix
- [X] Algorithm/Module
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

### Related issues
X

######################################

#### Module/Algorithm
- [ ] Core algorithm/module implementation
- [ ] [Query module](https://memgraph.com/docs/memgraph/reference-guide/query-modules) implementation
- [ ] Tests provided (unit / e2e)
- [ ] Code documentation
- [ ] README short description

### Documentation checklist
- [x] Add the documentation label tag
- [x] Add the bug / feature label tag
- [x] Add the milestone for which this feature is intended
    - If not known, set for a later milestone
- [X] Write a release note, including added/changed clauses
    - **Added a migration module from Postgres to Memgraph**
- [ ] Link the documentation PR here
    - **[Documentation PR link]**
- [X] Tag someone from docs team in the comments @kgolubic

### CODE REVIEW

File: python/migrate.py [hunk 1]
```diff
@@ -264,6 +266,85 @@ def cleanup_migrate_oracle_db():
...
+    rows = cursor.fetchmany(Constants.BATCH_SIZE)
+
+    return [mgp.Record(row=_name_row_cells(row, column_names)) for row in rows]
+
+
+def cleanup_migrate_postgresql():
+    global postgres_dict
+    postgres_dict[threading.get_native_id][Constants.CURSOR] = None
+    postgres_dict[threading.get_native_id][Constants.CONNECTION].close()
+    postgres_dict[threading.get_native_id][Constants.CONNECTION].commit()
```
Comment (antoniofilipovic) [inline python/migrate.py:340]: Can you revert these two, I think Buda correctly noticed once that we need first to commit and then close, this was bug

**Review by antoniofilipovic — CHANGES_REQUESTED** [review]
Just one change

**Review by antoniofilipovic — APPROVED** [review]
LGMT

**Comment by Josipmrden** [comment]
@kgolubic  I still need to add documentation, pls remind if I don't create it

**Comment by kgolubic** [comment]
@Josipmrden reminder for the docs.

**Comment by kgolubic** [comment]
I will use this as Release note:

- Added a PostgreSQL migration module to MAGE that enables direct migration from
  a PostgreSQL database to Memgraph. This new module streamlines the data import
  process by reducing the need for intermediate steps, such as using CSV files.
  Users can now migrate tables, including nodes and relationships, directly into
  Memgraph. 
  [#464](https://github.com/memgraph/mage/pull/464)

**Pull request merged**